stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -1222,6 +1222,7 @@ function factorize(A::StridedMatrix{T}) where T
     end
     qrfact(A, Val(true))
 end
+factorize(A::Adjoint) = adjoint(factorize(parent(A)))
```

Comment:
Why prefer `adjoint(factorize(parent(A)))` over `factorize(copy(A))`? Prefer avoiding an allocation to generating an object that may dispatch more nicely downstream?

Reply:
As you say, it saves an allocation and I think this within the core business of `Adjoint` since the main thing to do after `factorize` is solving and most solves ought to have methods for `Adjoint`.